lib/project_types/script/config/extension_points.yml

```diff
@@ -27,3 +27,16 @@ tax_filter:
     package: "@shopify/extension-point-as-tax-filter"
     sdk-version: "^9.0.0"
     toolchain-version: "^5.0.0"
+payment_methods:
+  domain: 'checkout'
```

Comment:
I could've extracted the _domain_ from the package name, but I don't see a hard reason to that instead of clearly defining it as I did here.

Reply:
Agreed that this is better :+1: